src/app/pages/press/inquiries/inquiries.js

```diff
@@ -1,16 +1,59 @@
-import {Controller} from 'superb.js';
-import {description as template} from './inquiries.html';
-import css from './inquiries.css';
+import PageContext from '../page-context';
+import React, {useContext} from 'react';
+import './inquiries.css';
+import {FontAwesomeIcon} from '@fortawesome/react-fontawesome';
 
-export default class Inquiries extends Controller {
+function FABrandIcon({icon}) {
+    return (
+        <FontAwesomeIcon icon={['fab', icon]} />
+    );
+}
+
+export default function Inquiries() {
+    const pageData = useContext(PageContext);
 
-    init(model) {
-        this.template = template;
-        this.model = model;
-        this.view = {
-            classes: ['inquiries']
-        };
-        this.css = css;
+    if (!pageData) {
+        return null;
     }
 
+    const name = pageData.press_inquiry_name;
+    const phone = pageData.press_inquiry_phone;
+    const email = pageData.press_inquiry_email;
+    const pressKitUrl = pageData.press_kit_url;
+
+    return (
+        <div className="inquiries">
+            <div>
+                <h2>Press inquiries</h2>
+                <div className="contact">
+                    {
+                        Boolean(name) && <div>{name}</div>
+                    }
+                    <div>{phone}</div>
+                    <div>{email}</div>
+                </div>
```

Comment:
should you make phone and email be a link with the scheme of `tel:` and `email:`  ?

Reply:
Good idea!